### PR TITLE
Cp fix 5

### PR DESCRIPTION
**Pull request by stephamon**
JIRA 2694

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4e3c3fdba7/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9a8553221a/how-to/)

File: content/rackspace-cdn/rackspace-cdn-edge-rules.md [hunk 1]
```diff
@@ -65,25 +65,23 @@ The default caching rules listed in the preceding section apply to all origins.
...
 
    <img src="{% asset_path rackspace-cdn/rackspace-cdn-edge-rules/Screen%20Shot%202015-10-02%20at%201.43.49%20PM.png %}" width="435" height="181" />
 
    **Note**: Verify that the origin can accept traffic from the domain listed in your CDN service. For example, if your CDN service has a domain of **cdn.customer1.com**, you might need to alter your origin server's **.htaccess** file to allow traffic for that domain.
 
-5. Tell the service when to fetch content from the new origin by specifying a content path. For example, i fyou want all of the images requested from the new origin, set the path to **/images/\***. With this rule, any time your your domain gets a request that contains **/images/** in the path, the CDN fetches that content from the new origin if it can't be found on the edge.
+6. Tell the service when to fetch content from the new origin by specifying a content path. For example, i fyou want all of the images requested from the new origin, set the path to **/images/\***. With this rule, any time your your domain gets a request that contains **/images/** in the path, the CDN fetches that content from the new origin if it can't be found on the edge.
```
Comment (RackerKate) [inline content/rackspace-cdn/rackspace-cdn-edge-rules.md]: "...For example, if you want..."

File: content/rackspace-metrics/create-a-grafana-dashboard-for-rackspace-metrics.md [hunk 2]
```diff
@@ -109,7 +109,7 @@ and a `check ID` to guarantee the uniqueness of the metrics. You can use the
 want to see.
 
 You can locate the entity ID and check ID in the [Cloud Control
-Panel](http://mycloud.rackspace.com/) on the Rackspace Intelligence
+Panel](http://login.rackspace.com/) on the Rackspace Intelligence
```
Comment (RackerKate) [inline content/rackspace-metrics/create-a-grafana-dashboard-for-rackspace-metrics.md]: https

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9ff0577e56/how-to/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-21bf16d1b3/how-to/)

**Pull request merged**